### PR TITLE
Handle generation for Julia+Stdlib

### PR DESCRIPTION
**Pull request by vdayanand**

### CODE REVIEW

File: src/worker_work.jl [hunk 1]
```diff
@@ -78,13 +78,24 @@ function package_docs(uuid, name, url, version, buildpath)
...
-            Pkg.activate(envdir)
-            doctype, rootdir = create_docs(pspec, buildpath)
-            meta["doctype"] = string(doctype)
-            meta["installs"] = true
-            @info("Done generating docs for $name")
-            monkeypatchdocsearch(uuid, name, buildpath)
-            package_source(uuid, name, rootdir, buildpath)
+             if name == "julia"
+                mktempdir() do path
+                    run(`wget --quiet -O $(joinpath(path, uuid*".tar.gz")) https://github.com/JuliaLang/julia/releases/download/v$(version)/julia-$(version).tar.gz`)
```
Comment (pfitzseb) [inline src/worker_work.jl]: Could we just use `download` here?

File: src/worker_work.jl [hunk 2]
```diff
@@ -78,13 +78,24 @@ function package_docs(uuid, name, url, version, buildpath)
...
+                    src_path = joinpath(path, name*"-"*version)
+                    cp(docs_path , buildpath, force=true)
+                    cp(src_path, joinpath(buildpath, "_packagesource") ,  force=true)
+                end
+            else
+                Pkg.activate(envdir)
+                doctype, rootdir = create_docs(pspec, buildpath)
+                meta["doctype"] = string(doctype)
+                meta["installs"] = true
+                monkeypatchdocsearch(uuid, name, buildpath)
```
Comment (pfitzseb) [inline src/worker_work.jl]: We presumably should do this for Julia too.

File: src/worker_work.jl [hunk 3]
```diff
@@ -123,8 +134,13 @@ function package_metadata(uuid, name, url, version, buildpath)
     try
         gh_auth = authenticate(readchomp(GIT_TOKEN_FILE))
         matches = match(r".*/(.*)/(.*\.jl)(?:.git)?$", url)
-        repo_owner = matches[1]
-        repo_name = matches[2]
+        if isnothing(matches) && name == "julia"
+            repo_owner = "Julialang"
```
Comment (pfitzseb) [inline src/worker_work.jl]: Proper capitalization: `JuliaLang` .

File: src/worker_work.jl (hunk 1 above)
Comment (vdayanand) [reply src/worker_work.jl]: wasn't aware of that function. Will do

File: src/worker_work.jl (hunk 3 above)
Comment (vdayanand) [reply src/worker_work.jl]: Done

File: src/worker_work.jl (hunk 2 above)
Comment (vdayanand) [reply src/worker_work.jl]: Done

**Pull request merged**